packages/gatsby-theme-api-docs/src/components/description.js

```diff
@@ -18,7 +22,11 @@ const DescriptionTextContainer = styled.span`
 // a table, card or bullet list.
 export const DescriptionText = (props) => (
   <DescriptionTextContainer data-search-key="embedded-api-description">
-    {markdownFragmentToReact(props.markdownString, {}, transformURNLinksPlugin)}
+    {markdownFragmentToReact(
+      props.markdownString,
+      { a: GatsbyLink },
```

Comment:
The "magic" gatsby link now has to be injected into the fragement renderer because the UI Kit has no understanding of gatsby or the app around it (which is the whole point of a separate UI kit)